knative-operator/deploy/resources/knativekafka/kafkasource-v0.17.1.yaml

```diff
@@ -445,7 +444,7 @@ metadata:
   name: kafka-source-webhook-certs
   namespace: knative-sources
   labels:
-    contrib.eventing.knative.dev/release: "v0.17.4"
```

Comment:
should we really replace the release label ?

Reply:
@matzew They don't exist in https://github.com/openshift/knative-eventing-contrib/blob/release-v0.17.1/openshift/release/knative-eventing-kafka-contrib-ci.yaml

Need manual update?